dataverse-persistence/src/main/resources/Bundle_en.properties

```diff
@@ -215,7 +215,7 @@ contact.dataset.header=Email Dataset Contact
 contact.to=To
 contact.support=Support
 contact.sender=Sender's E-mail
-contact.sender.required=User email is required.
+contact.sender.required=Sender's E-mail is required.
```

Comment:
Since those are not what is expected in https://github.com/CeON/dataverse/issues/1646 I'm guessing requirements were changed?

Reply:
Yes i changed them to adjust to other fields and Wojtek agreed.